backend/src/main/scala/cromwell/backend/validation/ValidatedRuntimeAttributesBuilder.scala

```diff
@@ -33,7 +34,15 @@ trait ValidatedRuntimeAttributesBuilder {
   /**
     * Returns a mapping of the validations: RuntimeAttributesValidation each converted to a RuntimeAttributeDefinition.
     */
-  final lazy val definitions: Seq[RuntimeAttributeDefinition] = validations.map(_.runtimeAttributeDefinition)
+  final lazy val definitions: Seq[RuntimeAttributeDefinition] =
+    validations.map(_.runtimeAttributeDefinition) ++ validations
+      .map {
+        case value: TwoKeyRuntimeAttributesValidation[_, _] =>
+          value.altKeyRuntimeAttributeDefinition
+        case _ =>
+          null
```

Comment:
Scalaism: we try not to use `null` at all. An alternate way to do this is something like:
```
validations.flatMap {
  case value: TwoKeyRuntimeAttributesValidation[_, _] => Seq(value.altKeyRuntimeAttributeDefinition)
  case _ => Seq()
}
```